quickly profile build fails during BOM flattening

### Bug description

I think it's due to the additional `xsltLocation` added in https://github.com/apache/camel-quarkus/commit/4761e7d8213d6379feb0f1b61223e52fd89ef24a#diff-2cdf2d35793fae7cd8f70171d04de916f2a6c295b19f099a0aea50589be615b4.

Copying `allow-findbugs.xsl` only happens during the `full` profile:

https://github.com/apache/camel-quarkus/blob/66ff4b18c3262314a04f882fdacfad916000a47c/pom.xml#L784-L808

Hence with `-Dquickly` we see:

```
[INFO] -------------< org.apache.camel.quarkus:camel-quarkus-bom >-------------
[INFO] Building Camel Quarkus :: BOM 3.23.0-SNAPSHOT
[INFO]   from pom.xml
[INFO] --------------------------------[ pom ]---------------------------------
[INFO]
[INFO] --- clean:3.4.1:clean (default-clean) @ camel-quarkus-bom ---
[INFO] Deleting /Users/james/Projects/camel-quarkus-main/poms/bom/target
[INFO]
[INFO] --- enforcer:3.5.0:enforce (enforce-java-version) @ camel-quarkus-bom ---
[INFO] Rule 0: org.apache.maven.enforcer.rules.version.RequireJavaVersion passed
[INFO]
[INFO] --- remote-resources:3.3.0:process (process-resource-bundles) @ camel-quarkus-bom ---
[INFO] Preparing remote bundle org.apache.apache.resources:apache-jar-resource-bundle:1.7
[INFO] Copying 3 resources from 1 bundle.
[INFO]
[INFO] --- cq:4.17.5:flatten-bom (flatten-bom) @ camel-quarkus-bom ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.644 s
[INFO] Finished at: 2025-05-16T11:55:01+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.l2x6.cq:cq-maven-plugin:4.17.5:flatten-bom (flatten-bom) on project camel-quarkus-bom: Execution flatten-bom of goal org.l2x6.cq:cq-maven-plugin:4.17.5:flatten-bom failed: Could not open resource /Users/james/Projects/camel-quarkus-main/poms/bom/target/enforcer-rules/allow-findbugs.xsl -> [Help 1]
```

The obvious workaround is to not clean and reuse the file generated from the previous non-quickly build.

--- a/pom.xml
+++ b/pom.xml
@@ -198,7 +198,7 @@
         <maven-utils.version>0.1.0</maven-utils.version>
 
         <!-- Maven plugin versions (keep sorted alphabetically) -->
-        <cq-plugin.version>4.17.5</cq-plugin.version>
+        <cq-plugin.version>4.17.9</cq-plugin.version>
         <cyclonedx-maven-plugin-version>2.9.1</cyclonedx-maven-plugin-version>
         <build-helper-maven-plugin.version>3.6.0</build-helper-maven-plugin.version>
         <exec-maven-plugin.version>3.5.0</exec-maven-plugin.version>

